download-latest.sh

```diff
@@ -140,7 +140,7 @@ get_os() {
 get_archi() {
     architecture=$(uname -m)
     case "$architecture" in
-    'x86_64' | 'amd64')
+    'x86_64' | 'amd64' | 'arm64')
```

Comment:
I would rather download the armv8 binary for the M1 :) If a machine has an `arm64` architecture and is not a M1, it will not work.

Reply:
```suggestion
    'x86_64' | 'amd64')
```